lib/chemistrykit/cli/cli.rb

```diff
@@ -265,37 +202,37 @@ def rspec_config(config) # Some of these bits work and others don't
             end
             @sc.finish
           end
-          c.order = 'random'
-          c.default_path = 'beakers'
-          c.pattern = '**/*_beaker.rb'
-          c.output_stream = $stdout
-          c.add_formatter 'progress'
 
-          html_log_name = options[:parallel] ? "results_#{options[:parallel]}.html" : 'results_0.html'
-
-          c.add_formatter(ChemistryKit::RSpec::HtmlFormatter, File.join(Dir.getwd, config.reporting.path, html_log_name))
+          unless options[:all]
+            c.filter_run @tags[:filter] unless @tags[:filter].nil?
+            c.filter_run_excluding @tags[:exclusion_filter] unless @tags[:exclusion_filter].nil?
+          end
 
+          c.capture_log_messages
+          c.treat_symbols_as_metadata_keys_with_true_values = true
+          c.order = 'random'
+          c.output_stream = $stdout
           # for rspec-retry
-          c.verbose_retry = true # for rspec-retry
+          c.verbose_retry = true
           c.default_retry_count = config.retries_on_failure
 
-          # TODO: this is messy... there should be a cleaner way to hook various reporter things.
-          if config.concurrency == 1 || options['parallel']
-            junit_log_name = options[:parallel] ? "junit_#{options[:parallel]}.xml" : 'junit_0.xml'
-            c.add_formatter(ChemistryKit::RSpec::JUnitFormatter, File.join(Dir.getwd, config.reporting.path, junit_log_name))
-          end
+          c.add_formatter 'progress'
+          
+          html_log_name = "results.html"
+          Dir.glob(File.join(Dir.getwd, config.reporting.path, "results*")).each { |f| File.delete(f) }
+          c.add_formatter(ChemistryKit::RSpec::HtmlFormatter, File.join(Dir.getwd, config.reporting.path, html_log_name))
+
+          junit_log_name = "junit.xml"
+          Dir.glob(File.join(Dir.getwd, config.reporting.path, "junit*")).each { |f| File.delete(f) }
+          c.add_formatter(ChemistryKit::RSpec::JUnitFormatter, File.join(Dir.getwd, config.reporting.path, junit_log_name))
         end
       end
       # rubocop:enable MethodLength
 
-      def run_in_parallel(beakers, concurrency, tags, options)
-        unless options[:all]
-          tag_string = tags[:filter].nil? ? nil : '--tag ' + tags[:filter].map { |k, v| "#{k}:#{v}" }.join(' ')
-          exclude_tag_string = tags[:exclusion_filter].nil? ? nil : '--tag ~' + tags[:exclusion_filter].map { |k, v| "#{k}:#{v}" }.join(' ')
-        end
-        config_string = '--config=' + options['config']
-        args = %w(--type rspec) + ['-n', concurrency.to_s] + ['-o', "#{config_string} #{tag_string} #{exclude_tag_string} --beakers="] + beakers
-        ParallelTests::CLI.new.run(args)
+      def run_parallel(beakers, concurrency)
+        require 'parallel_split_test/runner'
```

Comment:
this is a weird one. If I put the require at the top of the file, and you are running locally or not in parallel, then it causes regular rspec to not run correctly.